insolar/genesis.go

```diff
@@ -54,6 +56,8 @@ const (
 	GenesisAmountMigrationDaemonMembers       = 10
 	GenesisAmountActiveMigrationDaemonMembers = 3
 
+	GenesisAmountMigrationDaemonContract = 10
```

Comment:
Do we really need both GenesisAmountMigrationDaemonMembers and GenesisAmountMigrationDaemonContract?

Reply:
delete value GenesisAmountMigrationDaemonContract and change on GenesisAmountMigrationDaemonMembers